site/content/docs/5.0/customize/optimize.md

```diff
@@ -24,21 +24,33 @@ For instance, assuming you're using your own JavaScript bundler like Webpack or
 ```js
 // Import just what we need
 
-// import "bootstrap/js/dist/alert";
-// import "bootstrap/js/dist/button";
-// import "bootstrap/js/dist/carousel";
-// import "bootstrap/js/dist/collapse";
-// import "bootstrap/js/dist/dropdown";
-import "bootstrap/js/dist/modal";
-// import "bootstrap/js/dist/popover";
-// import "bootstrap/js/dist/scrollspy";
-// import "bootstrap/js/dist/tab";
-// import "bootstrap/js/dist/toast";
-// import "bootstrap/js/dist/tooltip";
+// import 'bootstrap/js/dist/alert';
+// import 'bootstrap/js/dist/button';
+// import 'bootstrap/js/dist/carousel';
+// import 'bootstrap/js/dist/collapse';
+// import 'bootstrap/js/dist/dropdown';
+import 'bootstrap/js/dist/modal';
+// import 'bootstrap/js/dist/popover';
+// import 'bootstrap/js/dist/scrollspy';
+// import 'bootstrap/js/dist/tab';
+// import 'bootstrap/js/dist/toast';
+// import 'bootstrap/js/dist/tooltip';
 ```
 
 This way, you're not including any JavaScript you don't intend to use for components like buttons, carousels, and tooltips. If you're importing dropdowns, tooltips or popovers, be sure to list the Popper.js dependency in your `package.json` file.
 
+{{< callout info >}}
+### Exports
+
+Files in `bootstrap/js/dist` use **default export**, so if you want to use one of them you have to do the following:
+
+{{< highlight js >}}
+import Modal from 'bootstrap/js/dist/modal'
+
```

Comment:
Removing this newline should fix the issue, but I'm not sure for the long-term fix so that we can use line breaks here.

Reply:
To clarify, the issue must be because we use `markdownify` in the callout shortcode:

https://github.com/twbs/bootstrap/blob/c5966de27395a407f9a3d20d0eb2ff8e8fb7b564/site/layouts/shortcodes/callout.html#L9

But removing `markdownify` means we won't be able to use Markdown in the callouts, which isn't something we want. After having a closer look, I think we are calling the callout shortcode wrong. We should use `{{% callout ... %}}` which parses the content as Markdown and remove our `markdownify` usage.

I'll need to have a closer look to see if this affects other places and I'll make a PR.

Reply:
All right, I can't make it work with `{{% foo %}}` either; the new line is causing an HTML error.

```html
div class="bd-callout bd-callout-warning">
<h5 id="ensure-correct-role-and-provide-a-label">Ensure correct <code>role</code> and provide a label</h5>
<p>
  In order for assistive technologies (such as screen readers) to convey that a series of buttons is grouped, an appropriate <code>role</code> attribute needs to be provided. For button groups, this would be
  <code>role=&quot;group&quot;</code>, while toolbars should have a <code>role=&quot;toolbar&quot;</code>.
</p>
<p>
  In addition, groups and toolbars should be given an explicit label, as most assistive technologies will otherwise not announce them, despite the presence of the correct role attribute. In the examples provided here, we use
  <code>aria-label</code>, but alternatives such as <code>aria-labelledby</code> can also be used.
</p>
<div class="highlight">
  <pre class="chroma"><code class="language-js" data-lang="js"><span class="kr">import</span> <span class="nx">Modal</span> <span class="nx">from</span> <span class="s1">&#39;bootstrap/js/dist/modal&#39;</span>
<!-- this is wrong and caused by the newline -->
<p>
<span class="kr">const</span> <span class="nx">modal</span> <span class="o">=</span> <span class="k">new</span> <span class="nx">Modal</span><span class="p">(</span><span class="nb">document</span><span class="p">.</span><span class="nx">getElementById</span><span class="p">(</span><span class="s1">'myModal'</span><span class="p">))</span>
</code></pre>
<!-- these 2 are wrong -->
</div></p>
</div>

```

I still think our callout usage is wrong, so I'll try to look into that, at least.

Reply:
I tried the line break but it doesn't work 😟

Reply:
I still see the line break present. This should work:

```
{{< highlight js >}}
import Modal from 'bootstrap/js/dist/modal'
const modal = new Modal(document.getElementById('myModal'))
{{< /highlight >}}
```

Reply:
ok now it works, thanks! 👍

Reply:
Actually please try this without the `\`:

```
`\`\`js
import Modal from 'bootstrap/js/dist/modal'

const modal = new Modal(document.getElementById('myModal'))
\`\`\`
```

It works for me, and it's a better solution. I will add it in my TODO to clean this situation up later :)

Reply:
OK, NVM the triple backticks works but I need to remove `markdownify`. So let's leave this snippet as is until I have a look later.